tasks/task2.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+const port = 3000;
+
+app.get('/', (req, res) => {
+  res.send('Greeting from Server');
+});
+
+app.get('/data', (req, res) => {
+  res.send('Here is some data');
+});
+
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`);
+});
+
+fs.writeFile('./files/file.txt', '', (err) => {
+  if(err) throw err;
+});
+
+fs.readFile('./files/file.txt', 'utf-8', (err, data) => {
+  if(err) throw err;
```

Comment:
this function should read file, which means that numbers from 1-10 will arrive as data property.
You can show data in console terminal and delete file.